contracts/release1/build1/build-metadata-R1B1.json

```diff
@@ -0,0 +1,21 @@
+{
+  "ui": "",
+  "change": "Initial build.",
+  "pluginSetupABI": {
+    "prepareInstallation": {
+      "description": "The information required for the installation of build 1.",
+      "inputs": [
+        {
+          "name": "number",
+          "type": "uint256",
+          "description": "The inital number to be stored."
+        }
+      ]
+    },
+    "prepareUpdate": {},
```

Comment:
why not also add description & inputs here? specially description, expaining why there is no inputs.

Reply:
My thinking was that the absence of objects inside `"prepareUpdate": {}` indicates that there are no possible updates. 
If there are builds from which an update is possible, we add them as

Let's imagine we write the metadata for build 5
```json
    "prepareUpdate": {
      "1": {
        "description": "The information required for the update from build 1 to build 5.",
        "inputs": [
          {
            "name": "account",
            "type": "address",
            "description": "The inital account to be stored."
          }
        ]
      },
      "2": {
        //...
      },
      "4": {
        //...
      }
    },
```
In this example, you for some reason cannot upgrade from build 3 to build 5.

Do you think we are missing a top-level description for all the updates?